### PR TITLE
Feature/e UI 2987 caseworker

### PR DESCRIPTION
**Pull request by udaydenduluri33**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/app/containers/header/header.component.ts [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import { Component, OnInit, Output, EventEmitter } from '@angular/core';
+import { FeatureToggleService } from '@hmcts/rpx-xui-common-lib';
```
Comment (paul-graham) [inline src/app/containers/header/header.component.ts:2]: Doesn't look like this is actually used anywhere in the code...?

File: src/app/containers/header/header.component.ts [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
 import { Store, select } from '@ngrx/store';
 import { CookieService } from 'ngx-cookie';
-import { Observable } from 'rxjs';
+import { Observable, of } from 'rxjs';
```
Comment (paul-graham) [inline src/app/containers/header/header.component.ts:5]: Doesn't look like `of` is actually used anywhere in the code...?

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

File: src/case-worker-ref-data/case-worker-ref-data.routing.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { RouterModule, Routes } from '@angular/router';
+import { AuthGuard } from 'src/services/auth/auth.guard';
```
Comment (paul-graham) [inline src/case-worker-ref-data/case-worker-ref-data.routing.ts:2]: This should be a relative path, not from "src".

File: src/case-worker-ref-data/components/upload-info-details/upload-info-details.component.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+                <p class="govuk-body">
+                    You’ll now be able to assign tasks to your colleagues <a href="https://manage-case.platform.hmcts.net/" target="_">using the Manage cases application</a>.
+                </p>
+                <p class="govuk-body">
+                <a href="#" class="govuk-link"></a>
+                </p>
+            </div>
+        </div>
+    </main>
+</div>
```
Comment (paul-graham) [inline src/case-worker-ref-data/components/upload-info-details/upload-info-details.component.html:27]: Needs a new line at the end.

File: src/case-worker-ref-data/components/upload-info-details/upload-info-details.component.ts [hunk 5]
```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { CaseWorkerRefDataUploadResponse } from '../../models/case-worker-ref-data.model';
+
+@Component({
+    selector: 'app-prd-upload-info-detail',
```
Comment (paul-graham) [inline src/case-worker-ref-data/components/upload-info-details/upload-info-details.component.ts:6]: Indenting weirdness.

File: src/case-worker-ref-data/containers/case-worker-ref-home/case-worker-ref-home.component.ts [hunk 6]
```diff
@@ -0,0 +1,9 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'app-prd-case-worker-ref-data-home',
```
Comment (paul-graham) [inline src/case-worker-ref-data/containers/case-worker-ref-home/case-worker-ref-home.component.ts:4]: Indenting weirdness.

File: src/case-worker-ref-data/containers/case-worker-ref-home/case-worker-ref-home.html [hunk 7]
```diff
@@ -0,0 +1 @@
+<app-prd-case-worker-details></app-prd-case-worker-details>
```
Comment (paul-graham) [inline src/case-worker-ref-data/containers/case-worker-ref-home/case-worker-ref-home.html:1]: New line needed at the end.

File: test/ngIntegration/util/ccdCaseConfigUtil.js [hunk 8]
```diff
@@ -0,0 +1,8 @@
+class caseConfigUtil{
+
+    getWizardPages(caseConfig){
+        const wizardpages = [];
+        
+    }
+
+}
```
Comment (paul-graham) [inline test/ngIntegration/util/ccdCaseConfigUtil.js:8]: New line needed.

**Review by paul-graham — CHANGES_REQUESTED** [review]

File: src/app/containers/header/header.component.ts (hunk 1 above)
Comment (udaydenduluri33) [reply src/app/containers/header/header.component.ts:2]: Agreed, removed it now.

File: src/app/containers/header/header.component.ts (hunk 2 above)
Comment (udaydenduluri33) [reply src/app/containers/header/header.component.ts:5]: Agreed, removed it now.

File: src/case-worker-ref-data/case-worker-ref-data.routing.ts (hunk 3 above)
Comment (udaydenduluri33) [reply src/case-worker-ref-data/case-worker-ref-data.routing.ts:2]: Yeah. The VS code auto code generator does this. Changed it now.

File: src/case-worker-ref-data/components/upload-info-details/upload-info-details.component.html (hunk 4 above)
Comment (udaydenduluri33) [reply src/case-worker-ref-data/components/upload-info-details/upload-info-details.component.html:27]: This is not required for HTML files.

File: src/case-worker-ref-data/containers/case-worker-ref-home/case-worker-ref-home.html (hunk 7 above)
Comment (udaydenduluri33) [reply src/case-worker-ref-data/containers/case-worker-ref-home/case-worker-ref-home.html:1]: This is not required for HTML files.

File: test/ngIntegration/util/ccdCaseConfigUtil.js (hunk 8 above)
Comment (udaydenduluri33) [reply test/ngIntegration/util/ccdCaseConfigUtil.js]: Done

File: src/case-worker-ref-data/containers/case-worker-ref-home/case-worker-ref-home.component.ts (hunk 6 above)
Comment (udaydenduluri33) [reply src/case-worker-ref-data/containers/case-worker-ref-home/case-worker-ref-home.component.ts:4]: Fixed it

File: src/case-worker-ref-data/components/upload-info-details/upload-info-details.component.ts (hunk 5 above)
Comment (udaydenduluri33) [reply src/case-worker-ref-data/components/upload-info-details/upload-info-details.component.ts]: fixed it

**Review by paul-graham — APPROVED** [review]

**Comment by udaydenduluri33** [comment]
All the changes for the caseworker are now moved to 441 which can goto master
https://github.com/hmcts/rpx-xui-approve-org/pull/441

**Pull request closed without merging**